flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java

```diff
@@ -164,6 +167,18 @@ private static Configuration mergeHiveConf(Configuration hadoopConf, String hive
         newConf.addResource(configFile);
       }
     }
+    if (!Strings.isNullOrEmpty(hadoopConfDir)) {
```

Comment:
Nit: we keep adding an extra line after a block. This is also done in the PR we are merging here

Reply:
+1. this also applies to line 181 below.

Reply:
thanks for this suggestion, address it.